### PR TITLE
add docs meta descriptions

### PR DESCRIPTION
**Pull request by alanszlosek**
Adding docs meta description to help with SEO and site search.

https://app.asana.com/0/1128567758154570/1203715918035826/f

Fixes #

## Target Release

1.4.x

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

File: website/docs/language/settings/backends/configuration.mdx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 page_title: Backend Configuration - Configuration Language
+description: >-
+  Use the `backend` block to control where Terraform stores state. Learn about the available state backends, the backend block, initializing backends, partial backend configuration, changing backend configuration, and unconfiguring a backend.
```
Comment (ritsok) [inline website/docs/language/settings/backends/configuration.mdx:4]:
```suggestion
  Use the `backend` block to control where Terraform stores state. Learn about configuring and managing backends.
```

just an idea to shorten this one a bit.

File: website/docs/language/modules/develop/providers.mdx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 page_title: Providers Within Modules - Configuration Language
+description: >-
+  Use providers within Terraform modules. Learn about version constraints, aliases, implicit inheritance, and passing providers explicitly in Terraform modules.
```
Comment (ritsok) [inline website/docs/language/modules/develop/providers.mdx:4]:
```suggestion
  Use providers within Terraform modules. Learn about version constraints, aliases, implicit inheritance, and passing providers to Terraform modules.
```

is that still right?

File: website/docs/cli/state/recover.mdx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 page_title: Recovering from State Disasters - Terraform CLI
-descriptin: >-
-  Commands that allow you to restore state backups and override Terraform state
-  protections.
+description: >-
+  Learn how to restore state backups and override Terraform state protections to fix state disasters with the Terraform CLI.
```
Comment (ritsok) [inline website/docs/cli/state/recover.mdx:4]:
```suggestion
  Learn how to restore state backups and override Terraform state protections to fix state errors with the Terraform CLI.
```

File: website/docs/cli/cloud/migrating.mdx [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ---
 page_title: Initializing and Migrating to Terraform Cloud - Terraform CLI
+description: >-
+  Learn how to migrate from local state, state backends, or a remote backend to Terraform Cloud with the Terraform CLI.
```
Comment (ritsok) [inline website/docs/cli/cloud/migrating.mdx:4]:
```suggestion
  Learn how to use the Terraform CLI to migrate local or remote state to Terraform Cloud.
```

if still accurate

**Review by ritsok — APPROVED** [review]

File: website/docs/language/settings/backends/configuration.mdx (hunk 1 above)
Comment (alanszlosek) [reply website/docs/language/settings/backends/configuration.mdx:4]: Jeffrey's examples were verbose so they hit SEO keywords and site search terms (which is where these descriptions are used). It's awkward, but his suggestions seem to have merit.

**Pull request merged**